### DISCUSSION TITLE
RustDesk iOS - Add option to hide blue cursor when connecting to Android

### DISCUSSION THREAD

**Discussion by plia7** [rustdesk/rustdesk#7179] [General]
When I connect to an Android target using RustDesk iOS app version 1.2.2, a blue mouse cursor shows on the screen:

![B709A6A7-B239-46A0-80C0-ED869CE6CC3A](https://github.com/rustdesk/rustdesk/assets/134864072/be421d6d-1650-4a30-a37d-5141da796f00)

It wasn’t showing it in RustDesk iOS app version 1.1.10.

Could you please add a setting or option not to show it as it used to in the previous version? I use the touch screen and don’t need to see this blue mouse cursor all the time.

Thank you.

**Comment by plia7** [comment]
up

**Comment by rustdesk** [comment]
@fufesou I remember this has been removed in nightly build, please find the PR or commit.

**Reply by fufesou** [reply]
https://github.com/rustdesk/rustdesk/pull/5874

**Comment by plia7** [comment]
> @fufesou I remember this has been removed in nightly build, please find the PR or commit.

Thank you, do you also remember in case if they removed the keyboard icon hiding restriction for the nightly build?

https://github.com/rustdesk/rustdesk/discussions/7183

**Reply by fufesou** [reply]
I'm not impressed. It seems that this function has not been implemented.

**Reply by plia7** [reply]
> I'm not impressed. It seems that this function has not been implemented.

Is it possible to implement that for the keyboard? This is really good help with the latency and trying to input keystrokes.

Thank you.
